www/options.php

```diff
@@ -47,7 +47,7 @@
             'tooltip' => 'Randomizes the dungeon bosses that each dungeon has'],
         'miniboss' => ['label' => 'Miniboss shuffle', 'type' => ['default' => 'Normal', 'shuffle' => 'Shuffle', 'random' => 'Randomize'], 'arg' => '--miniboss', 'default' => 'default',
             'tooltip' => 'Randomizes the dungeon minibosses that each dungeon has'],
-        'goal' => ['label' => 'Goal', 'type' => ['8' => '8 instruments', '7' => '7 instruments', '6' => '6 instruments', '5' => '5 instruments', '4' => '4 instruments', '3' => '3 instruments', '2' => '2 instruments', '1' => '1 instrument', '0' => 'No instruments', '-1' => 'Egg already open', 'random' => 'Random instrument count', 'seashells' => 'Seashell hunt (20)'], 'default' => '8', 'arg' => '--goal',
+        'goal' => ['label' => 'Goal', 'type' => ['8' => '8 instruments', '7' => '7 instruments', '6' => '6 instruments', '5' => '5 instruments', '4' => '4 instruments', '3' => '3 instruments', '2' => '2 instruments', '1' => '1 instrument', '0' => 'No instruments', '-1' => 'Egg already open', 'random' => 'Random instrument count', 'open-4' => 'Random short game (0-4)', '5-8' => 'Random long game (5-8)', 'seashells' => 'Seashell hunt (20)'], 'default' => '8', 'arg' => '--goal',
             'tooltip' => 'Changes the goal of the game.|[1-8 instruments], number of instruments required to open the egg.|[No instruments] open the egg without instruments, still requires the ocarina with the balled of the windfish|[Egg already open] the egg is already open, just head for it once you have the items needed to defeat the boss.|[Seashell hunt] egg will open once you collected 20 seashells. Instruments are replaced by seashells and shuffled.'],
```

Comment:
Short/long random are not explained in the tooltip.

Reply:
Ah yes; since `random` wasnt explained either I assumed that was intentional to avoid clutter